coverage/src/BamView-WebComponent.js

```diff
@@ -250,7 +250,16 @@ class BamViewChart extends HTMLElement {
         const end = parseInt(this.endInput.value.trim());
         const chromosomeNumber = chromosome.replace('chr', '');
 
-        if (this.validateInput(chromosomeNumber, start, end)) {
+        // Validate chromosome number first
+        if (!this.isValidChromosome(chromosomeNumber)) {
+            alert('Invalid chromosome number');
+            return;
+        }
+
+        // Check if only the chromosome is provided and start/end are NaN
+        if (isNaN(start) && isNaN(end)) {
```

Comment:
This isn't a very typical use of `isNaN`. I recommend detecting earlier whether start/end are set. I think the best place is `this.endInput.value === ""`

Reply:
So the full code would look something like:

```javascript
let end;
if (this.endInput.value === "") {
  end = parseInt(this.endInput.value);
}

if (end !== undefined) {
}

Reply:
For this PR, the new code is not too much. I think the diff looks like a big change because I moved the function zoomToChromosome and returned it because I call it when the start and end inputs are empty.

Reply:
I am a little confused about the type of variable "end" if the condition 

> if (this.endInput.value === "")

 is met. Would the variable end be "NaN" after parseInt is called?
 
 Please check the change in the new commit and let me know if the code works.